editor/editor_inspector.cpp

```diff
@@ -459,7 +459,8 @@ void EditorProperty::update_reload_status() {
 		Ref<Script> scr = object->get_script();
 		Variant orig_value;
 		if (scr->get_property_default_value(property, orig_value)) {
-			if (orig_value != object->get(property)) {
+			Variant o = object->get(property);
```

Comment:
Please give the variable a more meaningful name.

Reply:
@akien-mga Changed as requested.